scripts/ensure_aws_secrets.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+current_dir=$( dirname "${BASH_SOURCE[0]}" )
+source "$current_dir/common-functions.sh"
+
+get_secret() {
+  aws secretsmanager get-secret-value --secret-id "$1" --output text --query SecretString
+}
+
+"$current_dir/ensure_aws_profiles.sh"
+require_dev_aws_session
+
+export "KIELITESTI_TOKEN"="$(get_secret "kielitesti-token")"; echo "KIELITESTI_TOKEN exported"
```

Comment:
shellcheck complains about using `export` like this, but we don't have shellcheck in our lint job at the moment so I can't really criticize you about it. 😄